content/intro-to-storybook/react/en/test.md

```diff
@@ -39,7 +39,7 @@ Let's see how it works by tweaking the background of the `Task` component.
 Start by creating a new branch for this change:
 
 ```bash
-git checkout -b change-task-background
+git switch -c change-task-background
 ```
```

Comment:
@juunone we probably can reach a bit of a compromise here. Instead of shifting this to what you're proposing, we probably add the switch command as a aside. Turning this into:

```md
Start by creating a new branch for this change:


git checkout -b change-task-background

<div class="aside">
If you're using a newer version of git, you can replace this command with <code>git switch -c change-task-background</code>
</div>
````

The reason behind this, is that we cannot know for sure when the reader has installed git or if it's keeping it updated and if we make this as the instruction to be used, people might run into issues.

To be honest with you my windows rig was running a outdated version for quite some time and during that time i contributed to this and other repos and didn't even bother to check for updates (i'm aware that it's a bad practice), it was up until recently that i've updated to a newer version.

Let me know if you're ok with it and we can include this into the other frameworks covered by the tutorial.

Reply:
@jonniebigodes Thanks for detail answer!
you right, maybe not all of people check for git version.
it wont work for readers on older git versions so..
your turning guide answer is satisfies for everyone.

thank you for your time and hope to it incldue other tutorials.
thank you